wrappers/swift/PrivacyGuardInitConfig.swift

```diff
@@ -0,0 +1,23 @@
+//
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+//
+
+public class PrivacyGuardInitConfig {
```

Comment:
Do we expect subclasses of this class? If not, we should mark this as `final` to prevent unexpected behaviors if someone chooses to subclass.

Reply:
It does have some instance method so it might be potentially subclassed in future. so keep it non-final.